pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -0,0 +1,481 @@
+
+import numpy as np
+from pgmpy.models import BayesianModel
+from pgmpy.factors import TabularCPD, TreeCPD, RuleCPD
+from collections import OrderedDict, namedtuple
+
+
+edge_type = namedtuple('edge_type', ('start', 'end'))
+
+
+class DynamicBayesianNetwork(BayesianModel):
```

Comment:
@palashahuja  This class will deal only with the representation of DBNs. So, don't add anything that we don't need in representing the network. Also we want to keep things modular. So, we don't need timeslice in most of the places where you have used. For representing a complete DBN we will only need the 0th and 1st timeslice and the cpds associated with it.